Utils/python/filterdecisionproducer_cfi.py

```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
 filterDecisionProducer = cms.EDProducer('FilterDecisionProducer',
-trigTagArg1  = cms.string('TriggerResults'),
-trigTagArg2  = cms.string(''),
+trigTagArg1  = cms.string('slimmedPatTrigger'),
```

Comment:
this change breaks 2016 compatibility, so please remove all 2016 scenarios and input file lists (including from README, unit tests)